packages/react-api/src/api/stats.js

```diff
@@ -0,0 +1,49 @@
+import { checkCredentials, makeCall } from './common';
+import { MAP_TYPES, API_VERSIONS } from '@deck.gl/carto';
+
+const mandatoryProps = ['source', 'column'];
+
+/**
+ * Execute a stats service request.
+ *
+ * @param { object } props
+ * @param { string } props.column -
+ * @param { object } props.source -
+ * @param { object= } props.opts - Additional options for the HTTP request
+ */
+export async function callStats(props) {
+  mandatoryProps.forEach((prop) => {
+    if (!props[prop]) throw new Error('callStats: Missing source');
+  });
```

Comment:
for missing `column` it complains 'missing source'

Anyway, i would propose that "explicit" is more readable, more performant and ... even shorter:

```
assert(props.source, 'callStats: missing source')
assert(props. column, 'callStats: missing column')
```